### DISCUSSION TITLE
Android APP

### DISCUSSION THREAD

**Discussion by Ihor59** [joshua-8/ESP32_easy_wifi_data#8] [Q&A] [answered]
What android APP could be used to receive data from ESP, for instance sensor data obtained by controller?

**Comment by joshua-8** [comment, marked as answer]
Hi, I'm glad you're interested in this library.
https://github.com/RCMgames/RCMDS/tree/Android_Mode 
is a https://processing.org/ program built to work with this library. Using Android mode in processing you can install it on your Android phone. You could ignore or remove any of the controls and just write an intererface to display values that you receive.
I would be happy to answer any further questions about how to use these programs.

**Comment by Ihor59** [comment]
Thank you, I understand the idea. For me, it is interesting to construct from those programs something looking like a Wi-Fi serial plotter. I guess I could choose from your examples' protocol of communication via Wi-Fi between Android and ESP

**Comment by joshua-8** [comment]
That sounds like a great project!

I'd be happy to help if you have any questions.
